GitUI/SpellChecker/EditNetSpell.cs

```diff
@@ -369,7 +369,7 @@ private void LoadDictionary()
 
         private void ToggleAutoCompletion()
         {
-            if (!AppSettings.ProvideAutocompletion)
```

Comment:
Observation: In most situations where DesignMode changes are done (like on line 350), there is a comment too. 
I first wanted to have a comment here too, but the property is called DesignMode, should be clear enough.